shared/run_diff.go

```diff
@@ -104,22 +104,28 @@ func GetResultsDiff(before map[string][]int, after map[string][]int, filter Diff
 				if !filter.Deleted {
 					continue
 				}
-				diff[test] = []int{resultsBefore[1], resultsBefore[1]}
+				diff[test] = []int{resultsBefore[0], resultsBefore[1] - resultsBefore[0], -resultsBefore[1]}
 			} else {
 				if !filter.Changed && !filter.Unchanged {
 					continue
 				}
-				passDiff := abs(resultsBefore[0] - resultsAfter[0])
+				improved, regressed, delta := 0, 0, resultsBefore[0] - resultsAfter[0]
+				if delta < 0 {
+					improved = abs(delta)
+				} else {
+					regressed = delta
+				}
 				countDiff := abs(resultsBefore[1] - resultsAfter[1])
-				changed := passDiff != 0 || countDiff != 0
+				changed := delta != 0 || countDiff != 0
```

Comment:
Given that `countDiff` now isn't used for anything else, can't this be simplified to just `changed := resultsAfter != resultsBefore`? This does end up collapsing to just that, but it took me a while to tell.

Reply:
Done.